app/views/step_by_step_pages/show/_actions.erb

```diff
@@ -59,6 +59,10 @@
       <%= preview_link %>
     <% end %>
 
+    <% if @step_by_step_page.status.approved_2i? %>
+      <%= link_to 'Revert to draft', { controller: "review", action: "revert_to_draft", step_by_step_page_id: @step_by_step_page.id }, method: :post, data: { confirm: 'This will remove exisitng 2i approval. Do you want to revert to draft?' }, class: %w(govuk-link govuk-link--no-visited-state app-link--destructive) %>
```

Comment:
```suggestion
      <%= link_to 'Revert to draft', { controller: "review", action: "revert_to_draft", step_by_step_page_id: @step_by_step_page.id }, method: :post, data: { confirm: 'This will remove existing 2i approval. Do you want to revert to draft?' }, class: %w(govuk-link govuk-link--no-visited-state app-link--destructive) %>
```